[2024-06-22T14:29:40.347+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream scheduled__2024-06-22T14:23:30.090772+00:00 [queued]>
[2024-06-22T14:29:40.350+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream scheduled__2024-06-22T14:23:30.090772+00:00 [queued]>
[2024-06-22T14:29:40.351+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2024-06-22T14:29:40.356+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): kafka_data_stream> on 2024-06-22 14:23:30.090772+00:00
[2024-06-22T14:29:40.359+0000] {standard_task_runner.py:57} INFO - Started process 61557 to run task
[2024-06-22T14:29:40.361+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'kafka_spark_dag', 'kafka_data_stream', 'scheduled__2024-06-22T14:23:30.090772+00:00', '--job-id', '2028', '--raw', '--subdir', 'DAGS_FOLDER/dag_kafka_spark.py', '--cfg-path', '/tmp/tmprc3qja1m']
[2024-06-22T14:29:40.362+0000] {standard_task_runner.py:85} INFO - Job 2028: Subtask kafka_data_stream
[2024-06-22T14:29:40.377+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 2028 for task kafka_data_stream (DagRun for kafka_spark_dag with run_id or execution_date of 'scheduled__2024-06-22T14:23:30.090772+00:00' not found; 61557)
[2024-06-22T14:29:40.395+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-06-22T14:29:40.407+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
